cmd/openshift-install/agent/waitfor.go

```diff
@@ -37,6 +79,14 @@ func newWaitForInstallCompleteCmd() *cobra.Command {
 
 }
 
-func runWaitForInstallCompleteCmd() error {
-	return agentcmd.WaitFor()
+func runWaitForClusterValidationSuccessCmd(directory string) error {
```

Comment:
This was a misunderstanding I had of the tasks. I thought that because there was a third story we would make a third command. 😅 But the plan is to put the validation checks inside the bootstrap command.